clients/unshaded/src/main/java/tachyon/client/file/options/SetStateOptions.java

```diff
@@ -45,6 +46,17 @@ public Builder setPinned(boolean pinned) {
     }
 
     /**
+     * @param ttl the ttl(time to live) value (in milliseconds) to use; it specifies how long should
+     *            the file be deleted after creation, if it is set to {@link Constants#NO_TTL} and
```

Comment:
this indentation of JavaDoc comment is inconsistent with indentation we use elsewhere (e.g. setPinned)